erc/ERCS/erc-7562.md

```diff
@@ -342,17 +350,17 @@ These are storage rules that help to protect the bundler from denial of service:
 * **[EREP-030]** A Staked Account is accountable for failures in other entities (`paymaster`, `aggregator`) even if they are staked.
 * **[EREP-040]** An `aggregator` must be staked, regardless of storage usage.
 
-### Unstaked Entities Reputation Rules
+### Unstaked Paymasters Reputation Rules
 
 * Definitions:
     * **`opsSeen`, `opsIncluded`, and reputation calculation** are defined above.
     * `UnstakedReputation` of an entity determines the maximum number of entries using this entity allowed in the mempool.
     * `opsAllowed` is a reputation-based calculation for an unstaked entity, representing how many `UserOperations` it is allowed to have in the mempool.
-* **[UREP-010]** An unstaked sender is only allowed to have `SAME_SENDER_MEMPOOL_COUNT` `UserOperation`s  in the mempool. A staked sender is only limited by the SREP rules.
-* **[UREP-020]** For other entities: \
-  `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + (inclusionRate * INCLUSION_RATE_FACTOR) + (min(opsIncluded, 10000)`.
-    * This is a default of `SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT` for a new entity
-* **[UREP-030]** If an unstaked entity causes an invalidation of a bundle, its `opsSeen` is set to `1000`, effectively blocking it from inclusion for 24 hours.
+  * Rules:
+    * **[UREP-010]** An unstaked sender is only allowed to have `SAME_SENDER_MEMPOOL_COUNT` `UserOperation`s  in the mempool. A staked sender is only limited by the SREP rules.
+    * **[UREP-020]** For an unstaked paymaster only that is not throttled/banned: \
+      `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + inclusionRate * min(opsIncluded, MAX_OPS_ALLOWED_UNSTAKED_ENTITY)`.
+        * This is a default of `SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT` for a new paymaster
```

Comment:
Better add 
```
    * **[UREP-030]** unstaked factory has no limitation on mempool or bundle inclusion.
```
Since it is not clear from the text above.

(it is also implied that opsSeen/opsIncluded are required only for unstaked paymasters, not other unstaked entities)

Reply:
The problem is line 325, which says `All rules apply to all entities unless specified otherwise.`

I think it should only include staked entities and unstaked paymasters.  No reason to keep these numbers for unstaked factories and accounts, since they don't use a shared resource (like paymaster's deposit).

Reply:
Fixed